src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -298,7 +298,7 @@ private function getMetadata(string $key): array
             return [];
         }
 
-        return unserialize($entries);
+        return unserialize($entries) ?: [];
```

Comment:
`unserliaze` can return various types (see https://www.php.net/manual/en/function.unserialize.php#refsect1-function.unserialize-returnvalues).

Maybe we should test the return value with `is_array` ?

Reply:
Well, the Store controls the serialization too. So if unserializing does not fail, we know that we have an array (unless you messed up with the files directly to change what is stored, but that's not supported by our implementation)